sim/src/main/scala/TargetConfigs.scala

```diff
@@ -5,22 +5,24 @@ import freechips.rocketchip.tile._
 import freechips.rocketchip.tilelink._
 import freechips.rocketchip.subsystem._
 import freechips.rocketchip.devices.tilelink.BootROMParams
-//import boom.system.BoomTilesKey
+import boom.system.BoomTilesKey
 import testchipip.{WithBlockDevice, BlockDeviceKey, BlockDeviceConfig}
 import sifive.blocks.devices.uart.{PeripheryUARTKey, UARTParams}
 import icenet._
 
 class WithBootROM extends Config((site, here, up) => {
   case BootROMParams => BootROMParams(
-    contentFileName = s"./target-rtl/firechip/bootrom/bootrom.rv${site(XLen)}.img")
+    contentFileName = s"./target-rtl/firechip/testchipip/bootrom/bootrom.rv${site(XLen)}.img")
```

Comment:
Is the `firesim/sim/bootrom` directory still necessary? If not, we should remove it.

Reply:
@zhemao